build/run

```diff
@@ -121,13 +121,15 @@ if [ -z "${DISABLE_NESTED_DOCKER}" ]; then
     -v /var/run/docker.sock:/var/run/docker.sock"
 fi
 
+DOCKER_SERVER_API=$(docker version | grep 'API version' | tail -1 | awk '{print $3}')
```

Comment:
Sure. Docker version --format is much nicer. I'll change to that.

If docker is not running on the host the build fails anyway (at least it did for me). Maybe abort with a useful error message if docker version errors out?